src/storage/storage.rs

```diff
@@ -0,0 +1,27 @@
+use mongodb::{Client, options::ClientOptions, Collection, Database};
+
+use crate::error::AppError;
+
+use super::secret_word::SecretWord;
+
+pub struct Storage {
+    pub client: Client,
+    pub db: Database,
+    pub secret_word_collection: Collection<SecretWord>
+}
+
+impl Storage {
+    pub async fn new() -> Result<Self, AppError> {
+        let mut client_options = ClientOptions::parse("mongodb://localhost:27017").await?;
```

Comment:
Can we move this to be in the `Config` object?

I think we might want to change this in the future during deploy